src/state/endpoints/tests/reducer.test.js

```diff
@@ -0,0 +1,79 @@
+import deepfreeze from 'deepfreeze';
```

Comment:
Truthfully, I'd not have used `create-react-app` 😄  So I'd have done something more like what we have in Calypso: `test` directory with matching file name to parent directory.

I think it's fine as-is, `__tests__` is strange indeed.